test/test-untrack.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+. "test/testlib.sh"
+
+begin_test "untrack"
+(
+  set -e
+
+  # no need to setup a remote repo, since this test doesn't need to push or pull
+
+  reponame="untrack"
+  git init $untrack
+  cd $untrack
```

Comment:
`$untrack` is wrong, so this was running init and this untrack test in `$TRASHDIR`. If this test runs before the next one, the next one always fails because it doesn't expect this first test to init `$TRASHDIR`.

Fixed in b756d5611613b6eb55ae359dda3e8e86f7dac3d5